controller/src/main/java/org/jboss/as/controller/ControlledProcessState.java

```diff
@@ -195,7 +206,7 @@ public void revertRestartRequired(Object stamp) {
         }
     }
 
-    ControlledProcessStateService getService() {
+    public ControlledProcessStateService getService() {
```

Comment:
I don't see any uses of this outside the package so I think this can remain package protected.